app/src/main/res/layout/google_login.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be better to name this as `activity_login` seeing it's the main layout for LoginActivity. It's just convention. 
https://hungsonandroid.wordpress.com/2016/10/17/androidnaming-convention/